impress-org/givewp
# Refactor: update header image as a global template

## Description
This PR updates the header image as a global setting that is accessible for all layouts. It also removes the the HeaderTextWrapper &  subscriber settings that are & will no longer be used.

## Affects

FormDesignSettings -> HeaderImage

## Visuals

https://github.com/impress-org/givewp/assets/75056371/fcf5d687-7aca-474b-8db3-c41e1296131a


https://github.com/impress-org/givewp/assets/75056371/e4682bf2-2826-4c4c-95aa-6fd384090d8b


https://github.com/impress-org/givewp/assets/75056371/1a7af6a4-672d-4c23-94a5-ed37d86302ea


## Testing Instructions
Build: https://github.com/impress-org/givewp/actions/runs/7806011239
- Upload a header image
- Delete a header image
- Test the 3 layouts : above, center & background
- Remove header description & title to verify layout does not break.
- View on frontend & in builder

## Pre-review Checklist
